apis/shared-dimensions/lib/shapes/terms-query-shape.ttl

```diff
@@ -0,0 +1,102 @@
+@prefix sparql: <http://datashapes.org/sparql#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix hydra: <http://www.w3.org/ns/hydra/core#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix schema: <http://schema.org/> .
+@prefix meta: <https://cube.link/meta/> .
+@prefix s2q: <https://hypermedia.app/shape-to-query#> .
+
+[
```

Comment:
This is the shape which is the source for the query. The limit, offset, sh:filterShape and sh:expression are dynamically preprocessed to adjust for API query arguments